### PR TITLE
17029 - Expandable Help for Business Dashboard

### PR DESCRIPTION
**Pull request by ketaki-deodhar**
*Issue #:*
https://github.com/bcgov/entity/issues/17029

*Description of changes:*
Implement Expandable help. Below is the UXpin for reference https://preview.uxpin.com/a86dccdd20ac62828f965c5eea10f81b828a9853#/pages/163762254/comment/no-panels

Some screenshots:

image

image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/components/auth/manage-business/EntityManagement.vue [hunk 1]
```diff
@@ -858,7 +870,7 @@ export default class EntityManagement extends Mixins(AccountMixin, AccountChange
   justify-content: space-between;
 
   h1 {
-    margin-bottom: -10px;
+    margin-bottom: -30px;
```
Comment (seeker25) [inline auth-web/src/components/auth/manage-business/EntityManagement.vue:873]: `mb-n8` if it was 32px :(

File: auth-web/src/components/auth/manage-business/EntityManagement.vue [hunk 2]
```diff
@@ -871,6 +883,9 @@ export default class EntityManagement extends Mixins(AccountMixin, AccountChange
     font-weight: 700;
   }
 }
+.my-business-help {
+  margin-bottom: 60px;
```
Comment (seeker25) [inline auth-web/src/components/auth/manage-business/EntityManagement.vue:887]: `mb-15`?

Comment (ketaki-deodhar) [reply auth-web/src/components/auth/manage-business/EntityManagement.vue:892]: Let me try this and see if it works

**Comment by ketaki-deodhar** [comment]
@seeker25 could you please add Severin as Reviewer as well and me as assignee? Somehow I cannot add/edit anything

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2461-01u22yrn.web.app

**Comment by seeker25** [comment]
@ketaki-deodhar I've given you write access, should be able to do what you want now

**Pull request closed without merging**